docs/packages/markdown/parseMarkdown.js

```diff
@@ -286,7 +286,7 @@ function createRender(context) {
 
       return `<div class="MuiCode-root"><pre><code class="language-${escape(lang, true)}">${
         escaped ? code : escape(code, true)
-      }</code></pre><button data-ga-event-category="code" data-ga-event-action="copy-click" aria-label="Copy the code" class="MuiCode-copy">Copy <span class="MuiCode-copyKeypress"><span>(Or</span> $keyC<span>)</span></span></button></div>\n`;
+      }</code></pre><button data-ga-event-category="code" data-ga-event-action="copy-click" aria-label="Copy the code" class="MuiCode-copy">Copy <span class="MuiCode-copyKeypress"><span>(or</span> $keyC<span>)</span></span></button></div>\n`;
```

Comment:
It feels better lowercase:

**Before**
<img width="165" alt="Screenshot 2022-10-23 at 01 44 06" src="https://user-images.githubusercontent.com/3165635/197366376-428dcd61-39b9-438b-807b-dd59a5aad127.png">


**After**
<img width="132" alt="Screenshot 2022-10-23 at 01 44 53" src="https://user-images.githubusercontent.com/3165635/197366389-a4f81a0c-efc5-4609-ab1f-411dc8289b79.png">